web/src/admin/roles/RoleViewPage.ts

```diff
@@ -71,7 +72,7 @@ export class RoleViewPage extends AKElement {
         });
     }
 
-    render(): TemplateResult {
+    render() {
```

Comment:
any reason for removing the type annotations?

Reply:
Two reasons, actually: the first is that [Typescript generally recommends against it](https://www.typescriptlang.org/docs/handbook/2/functions.html), as inference of return type is very good and the only reason to do it is if you don't trust the function body itself to generate the correct return type, and the other is that Lit's re-render algorithm is smarter with `nothing` rather than `html\`\`` , but doesn't conform to TemplateResult without a lot of fussiness.